doc/specs/#6899 - Action IDs/#6899 - Action IDs.md

```diff
@@ -0,0 +1,228 @@
+---
+author: Mike Griese @zadjii-msft
+created on: 2020-07-13
+last updated: 2020-07-22
+issue id: 6899
+---
+
+# Action IDs
+
+## Abstract
+
+This document is intended to serve as an addition to the [Command Palette Spec],
+as well as the [New Tab Menu Customization Spec].

```

Comment:
todo links?

Reply:
These are actually footnote-style markdown links - see L218-219 to see where they're defined
![image](https://user-images.githubusercontent.com/18356694/90021550-0ae1a380-dc77-11ea-9f5b-2c3934431e98.png)

They're defined at the bottom of the doc, so if I want to re-use the same text as a hyperlink elsewhere in the doc, I don't have to reprint the URL, or even have the URL inline. 

Handy for specs like this IMO